tools/deploy.py

```diff
@@ -50,7 +50,12 @@
       reader = csv.reader(csvfile)
       for row in reader:
         print("en: "+row[0]+" "+LOCALE+": "+row[1])
-        subprocess.run("find . -name 'fingerprints.yml' | xargs sed -i '' 's/deviceLabel: "+row[0].translate(slash_escape)+"/deviceLabel: "+row[1].translate(slash_escape)+"/g'", shell=True)
+        subprocess.run(
+          "find . -name 'fingerprints.yml' | xargs sed -i '' 's/deviceLabel: "+row[0].translate(slash_escape)+"/deviceLabel: "+row[1].translate(slash_escape)+"/g'",
+          shell=True
+          cwd=os.path.dirname(current_path)
```

Comment:
explicitly setting the working directory is the only real thing that changed here